ui/src/components/virtual-scroll/use-virtual-scroll.js

```diff
@@ -691,7 +691,15 @@ export function useVirtualScroll ({
     setVirtualScrollSize()
   })
 
+  let shouldActivate = false
+
+  onDeactivated(() => {
+    shouldActivate = true
+  })
+
   onActivated(() => {
+    if (shouldActivate !== true) { return }
+
```

Comment:
Why is there a flag needed? onActivated() is always triggered only after an onDeactivated().